tokenserver/assignment/sqlnode/migrations/versions/8440ac37978a_migration.py

```diff
@@ -0,0 +1,23 @@
+"""migration
+
+Revision ID: 8440ac37978a
+Revises: 75e8ca84b0bc
+Create Date: 2019-12-16 15:46:07.048437
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '8440ac37978a'
+down_revision = '75e8ca84b0bc'
+
+
+def upgrade():
+    op.add_column("users", sa.Column("migration_state", sa.String))
```

Comment:
If you're adding columns here, please update the schema description in the README to match.